### DISCUSSION TITLE
Mentor Discussion

### DISCUSSION THREAD

**Discussion by sanjay-kv** [recodehive/Stackoverflow-Analysis#404] [General]
https://github.com/recodehive/Stackoverflow-Analysis/pull/403#issuecomment-2418358672

@shristirwt this will be our discussion place

**Comment by sanjay-kv** [comment]
@shristirwt Please watch this video 
https://www.loom.com/share/ec4bf5af088a43a7a19fda67c33a0c94?sid=7c2e0b0c-5d84-4595-a728-1f762dd45467

**Reply by shristirwt** [reply]
@sanjay-kv Thank you so much for providing me this opportunity and helping me grow 
I watched your video and I will be consistent with the repo and try my best as a maintainer

**Reply by sanjay-kv** [reply]
welcome to the team. i will connect with you on weekend to provide more info

**Comment by sanjay-kv** [comment]
your current profile: im just keeping ss so u can come back later and see what all changed
![screencapture-github-shristirwt-2024-10-17-14_42_51](https://github.com/user-attachments/assets/9f8f938a-90d0-4ea6-bd74-1455909cf45c)
Organisation to join:
AccessibleForAll: https://github.com/AccessibleForAll
Devs-Dungeon: https://github.com/Devs-Dungeon
Magic-Academy: https://github.com/Magic-Academy
EddieHubCommunity: https://github.com/EddieHubCommunity
Bauddhik-Geeks: https://github.com/Bauddhik-Geeks
github-craftwork: https://github.com/github-craftwork

Signin up for developer program member: https://github.com/settings/profile#github-developer-program

apply for students developer pack: https://education.github.com/pack

**Reply by shristirwt** [reply]
While registering for GitHub developer program member what should I enter in product or company website

![Screenshot 2024-10-17 113249](https://github.com/user-attachments/assets/e3450ffd-5889-4dd7-87ce-e54d3adf5586)

**Comment by shristirwt** [comment]
@sanjay-kv The app is running locally without any error 
I tried to search it on streamlit cloud manually but I could not find it 
There was an option of Deploy when I ran it locally
Do you have any suggestions regarding it which can help me out 

![Screenshot 2024-10-17 161226](https://github.com/user-attachments/assets/cf013f66-a488-45b2-9228-c705bc5566c7)
![Screenshot 2024-10-17 161239](https://github.com/user-attachments/assets/78ade5f1-c882-4569-a7de-8485a2767dea)

**Reply by sanjay-kv** [reply]
i tried to work on this, maybe we can raise new issue on this, so other contributor who are good at it can help to deploy

**Comment by sanjay-kv** [comment]
![image](https://github.com/user-attachments/assets/b7c8755f-cca9-46a2-a899-d6490204873f)

@shristirwt limit to only 2 repos.

**Reply by sanjay-kv** [reply]
![image](https://github.com/user-attachments/assets/a0f3f1e8-335a-499e-b5a1-db3a1da3f363)
activate the graph

![image](https://github.com/user-attachments/assets/f4d7df15-5a2d-4f85-8fb0-d650fa19a08b)

enable activity overview

**Reply by shristirwt** [reply]
I have made the changes accordingly

**Comment by sanjay-kv** [comment]
Signin up for developer program member: https://github.com/settings/profile#github-developer-program

https://github.com/recodehive/Support/issues/new?assignees=&labels=invite+me+to+the+community&projects=&template=invitation.yml&title=Please+invite+me+to+the+Recode-Hive+GitHub+Community+Organization

join recodehive org. 
next task for you @shristirwt

**Reply by shristirwt** [reply]
What do I need to enter here 

![Screenshot 2024-10-17 224204](https://github.com/user-attachments/assets/80d8790a-29bd-4333-aee2-8678df856b5e)

**Reply by sanjay-kv** [reply]
i dont think thats mandtory, you can skip it

**Reply by shristirwt** [reply]
I guess I need to fill it 

![Screenshot 2024-10-18 185908](https://github.com/user-attachments/assets/3a120229-2bf3-4d87-a19f-37caf7878773)

**Reply by sanjay-kv** [reply]
you can use www.recodehive.com @shristirwt

**Comment by Akash-Gupta-git** [comment]
Can we fill any of our website in product or company website ?
while filling Gitthub Developer Program

**Reply by sanjay-kv** [reply]
yes

**Comment by sanjay-kv** [comment]
@shristirwt can you remove the unencessary file to backlog folder

**Reply by sanjay-kv** [reply]
![image](https://github.com/user-attachments/assets/1a54a090-1cf1-4fbf-8e60-43d23dadcd48)


website is not working, have alook into it as well.

**Reply by shristirwt** [reply]
@sanjay-kv You advised me that we can raise an issue on this so that other contributors can work upon it.
Shall I raise an issue ?

**Reply by sanjay-kv** [reply]
sure, you are maintainer now, can raise and make actions. just tag me in the issue so i get the info

**Reply by shristirwt** [reply]
> sure, you are maintainer now, can raise and make actions. just tag me in the issue so i get the info

sure sir

**Comment by shristirwt** [comment]
@sanjay-kv I raised the issue #412 . Please have a look on it and add the label as required. Let me know if further changes are required.

And sir if I want to contribute then I can assign the issue to me by myself or you will assign it to me as there are levels which are to be labeled.
I have raised #413 to contribute

**Reply by sanjay-kv** [reply]
dont assign labels on your PR, other contributor will create issue

**Comment by shristirwt** [comment]
@sanjay-kv Sir I have setup my sponsors profile. I was working on enabling my sponsor button but I could not understand much. I have created a FUNDING.yml file but for now the sponsor button has not been enabled

**Comment by shristirwt** [comment]
@sanjay-kv Sir I have setup my sponsors profile . Please have a  look into it and let me know if any changes are required.

**Reply by sanjay-kv** [reply]
its awesome.

**Comment by shristirwt** [comment]
@sanjay-kv Sir I have made a pull request that has not been merged and an issue that has not been assigned

**Comment by shristirwt** [comment]
@sanjay-kv Sir for Hack Web3Conf can I submit Stackoverflow-Analysis as a project 

Also sir, I have made a PR on Stackoverflow-Analysis which has not been merged.

And sir, Can I review code of other contributors?

**Reply by sanjay-kv** [reply]
yes you can review. 
just dont merge it.

**Reply by sanjay-kv** [reply]
you can submit stack overflow

**Reply by shristirwt** [reply]
Thank you sir

**Comment by shristirwt** [comment]
@sanjay-kv Sir I had asked you if I could submit your repository Stackoverflow-Analysis for Hack web3conf and you permitted me to do so. I submitted it and raised a PR but the maintainers are asking me that you might have misunderstood it and it is not permissible to do so. Can you confirm it?

**Reply by sanjay-kv** [reply]
i will check, im not much sure about the criteria of it. i will check n confirm

**Reply by shristirwt** [reply]
Sure sir

**Comment by shristirwt** [comment]
@sanjay-kv Sir for Stackoverflow-Analysis repo we don't have survey_results_public_2021,2022 and 2023 neither in the repo nor in the drive link

**Comment by shristirwt** [comment]
@sanjay-kv 
Can you have a look upon issue #518 
Is it related to the repo by anyhow? or should I close it

**Reply by sanjay-kv** [reply]
@shristirwt you can ignore this, its just people trying to get the badge

**Reply by shristirwt** [reply]
yes sir

**Comment by CodeWithMehru** [comment]
Please give me a star to achieve the badge & I will also give you star on your repository.
https://github.com/CodeWithMehru/MehruXploit
